tests/Fhi.HelseIdSelvbetjening.CLI.Tests/AcceptanceTests/AcceptanceTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Fhi.HelseIdSelvbetjening.CLI.Commands.GenerateKey;
+using Fhi.HelseIdSelvbetjening.CLI.Commands.UpdateClientKey;
+
+namespace Fhi.HelseIdSelvbetjening.CLI.AcceptanceTests
+{
+    /// <summary>
+    /// Manual acceptance tests for the CLI. These tests should be runned against test environment.
+    /// </summary>
+    public class AcceptanceTests
+    {
+        /// <summary>
+        /// In order to run this test:
+        /// 1. Set directory to where new keys should be stored
+        /// 2. Set directory to where existing (old) keys is stored
+        /// 3. Set clientId
+        /// 
+        /// Note: In order to update secret the nhn:selvbetjening/client scope must be set on the client
+        /// </summary>
+        /// <returns></returns>
+        [Test]
+        [Ignore("This test generates keys and update client with new keys should be run manually.")]
```

Comment:
Bruk explicit istedet for Ignore.  Ignore indikerer noe som skal rettes opp, men som midlertidig er satt på vent.  Expliti indikerer at den må kjøres manuelt.

```suggestion
        [Explicit("This test generates keys and update client with new keys should be run manually.")]
```